contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -899,6 +906,60 @@ SetBulkLoadRowData(TDSRequestBulkLoad request, StringInfo message)
 	return message;
 }
 
+static void
+CleanupBCPDuringError(bool internal_sp_started,
+					 volatile int before_subtxn_id,
+					 volatile int before_lxid,
+					 ResourceOwner oldowner,
+					 MemoryContext oldcontext)
+{
+	int			ret = 0;
+
+	HOLD_CANCEL_INTERRUPTS();
```

Comment:
The cleanup function doesn't care whether you're pausing the interrupts or not. It's rather from where you're calling the function, it matters. In this case, since you're calling in catch block, you want to hold the interrupts. 
Yeah, the comment part looks redundant. Please ignore.